gradoop-common/src/main/java/org/gradoop/common/model/impl/properties/PropertyValue.java

```diff
@@ -117,10 +117,24 @@
   public static final transient byte OFFSET            = 0x01;
 
   /**
-   * We use a short as length prefix in binary representations of this property.
-   * This value is the maximum viable length.
+   * Bit flag indicating a "large" property. The length of the byte representation will be stored
+   * as an {@code int} instead.
+   */
+  public static final transient byte FLAG_LARGE = 0x10;
+
+  /**
+   * Was previously used to indicate maximum size of the internal byte representation.
+   *
+   * @deprecated The new maximum size is depending on the JVM implementation.
+   */
+  @Deprecated
+  public static final transient int MAX_BINARY_LENGTH = Short.MAX_VALUE - OFFSET;
```

Comment:
I wonder if we still need that at all. Iirc, it's only used in the validation, which is deleted now.